test/e2e/scenarios/add_cluster.go

```diff
@@ -0,0 +1,15 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Adding exisitng network policy to newly added cluster", func() {
+	PIt("Should implement exisitng network policy in newly added cluster", func() {
+		By("Create a new cluster")
+		By("Discover the newly created cluster")
+		By("creating pod1 with lable1 in cluster1 in namespace1")
```

Comment:
nit: label (including one in the next line)

Reply:
Done